drivers/docker/driver.go

```diff
@@ -624,6 +627,10 @@ func (d *Driver) containerBinds(task *drivers.TaskConfig, driverConfig *TaskConf
 			task.Env[taskenv.SecretsDir] = strings.ReplaceAll(string(s[2:]), "\\", "/")
 		}
 	}
+}
+
+func (d *Driver) containerBinds(task *drivers.TaskConfig, driverConfig *TaskConfig) ([]string, error) {
+	d.convertAllocPathsForWindowsLCOW(task, driverConfig.Image)
```

Comment:
I think the same issue as before - I wouldn't have expected `containerBinds` to mutate the inputs, even if indirectly.  I'd suggest moving the calling site of `d.convertAllocPathsForWindowsLCOW`  to `StartTask` , right after loading the image?

Reply:
Done.

I thought that it would be better to extract the runtime windows check to outside of the function, so that function is not executed if it's not windows.